packages/transactions/src/signatures.ts

```diff
@@ -46,7 +46,10 @@ export async function partiallySignTransaction<TTransaction extends CompilableTr
     const wireMessageBytes = getCompiledMessageEncoder().encode(compiledMessage);
     const publicKeySignaturePairs = await Promise.all(
         keyPairs.map(keyPair =>
-            Promise.all([getAddressFromPublicKey(keyPair.publicKey), signBytes(keyPair.privateKey, wireMessageBytes)]),
+            Promise.all([
+                getAddressFromPublicKey(keyPair.publicKey),
+                signBytes(keyPair.privateKey, wireMessageBytes as Uint8Array),
```

Comment:
You should be able to remove that cast on the next PR.

Reply:
And also kill this entire file 1,000,000 PRs into the future :D